README.md

```diff
@@ -29,17 +29,78 @@ To better communicate and more easily build an API and UI, we've decided on a co
 
 We are planning to use the following tools:
 
-- [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend
+- [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend using JavaScript/TypeScript
 - [Postgres](https://www.postgresql.org) with [Sequelize ORM](https://sequelize.org)
 - [Elasticsearch](https://www.elastic.co/what-is/elasticsearch)
-- A [React](https://reactjs.org/) frontend using JavaScript (not TypeScript) and CSS (not Sass)
+- A [React](https://reactjs.org/) frontend using JavaScript/TypeScript and CSS (not Sass)
+- [Next.js](https://nextjs.org/) for server-side rendering
 
 A lot of people know these tools, and they're proven to work well at scale.
 
 We are considering using a tool like [Next.js](https://nextjs.org) to get up and running faster.
 
 We will focus on building an open API first. Then developers can use the API to build their own mobile clients and voice interface clients.
 
+## Development Setup
+
+Requirements: Node.js, Docker, internet access
+
+### Installing Node.js
+
+Follow instructions for downloading and installing Node.js for your opreating system from the [official Node.js website](https://nodejs.org/en/download/).
+
+Ensure you are installing Node 10 or greater and npm 6 or greater.
+
+### Installing Docker
+
+Click [here](https://docs.docker.com/v17.12/install/) for the Docker installation site.  Scroll down to "Supported Platforms" and follow the instructions to download & install Docker Desktop for your operating system (or Docker CE for linux).
```

Comment:
```suggestion
Click [here](https://docs.docker.com/install/#supported-platforms) for the Docker installation site and follow the instructions to download & install Docker Desktop for your operating system (or Docker CE for linux).
```

17.12 is outdated, 19.03 is the current version. This link ☝️ always goes to the latest.